<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreastLesionSegmentation</class>
 <widget class="qMRMLWidget" name="BreastLesionSegmentation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>401</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image Selector</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qMRMLNodeComboBox" name="inputSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
          <string>vtkMRMLVectorVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Segmentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="startSegmentationButton">
        <property name="text">
         <string>Start Segmentation</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveMaskButton">
        <property name="text">
         <string>Save segmented mask</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>BreastLesionSegmentation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
